src/index.html

```diff
@@ -11,7 +11,207 @@
+          max="100"
+          >
+        </label>
+
+        <label class="form-field">
+            Full date of birth:
+          <input
+          type="date"
+          name="birthday"
+          >
+        </label>
+
+        <label class="form-field">
+            I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="terms"
+          required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="container">
+        <legend>
+        Registration
+        </legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          >
+        </label>
+        
+        <label class="form-field">
+          Password
+          <input
+          type="password"
+          name="password"
+          minlength="5"
+          maxlength="32"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="container">
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <label class="form-field">
+          Do you love cats?
+          <input
+              type="radio"
+              name="cats"
+              value="yes"
+          >
+            Yes
+          <input
+              type="radio"
+              name="cats"
+              value="no"
+          >
+            No
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+          type="color"
+          name="color"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+          type="time"
+          name="time"
+          step="1"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+          name="brand"
+          multiple
+          >
+            <option
+            value="bmw"
+            >
+              BMW
+            </option>
+
+            <option 
+            value="audi"
+            >
+              Audi
+            </option>
+
+            <option 
+            value="lada"
+            >
+              Lada
+            </option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input
+          type="range"
+          name="rate"
+          min="1"
+          max="10"
+          value="1"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="container">
+        <legend>
+          Additional info:
+        </legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea
+          name="comments"
+          id="comments"
+          >
+          </textarea>
+        </label>
+
+        <label class="form-field">
+          Would you recommend us?
+          <select
+          name="recommend"
+          >
```

Comment:
no need to use a new blank for 1 attribute
```suggestion
          <select name="recommend">
```